packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -97,12 +98,22 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
         }
     }
 
+    /**
+     * Get the categories for the selected venue.
+     */
+    function getVenueCategories(venue) {
+        mapsindoors.services.LocationsService.getLocations({}).then(locations => {
```

Comment:
Write a comment about us not being able to get the venue parameter to work here, which is why we do the filtering. Also create a ticket in the SDK project about that, with steps to reproduce.